.vscode/settings.json

```diff
@@ -17,4 +18,5 @@
         "editor.tabSize": 2,
         "editor.insertSpaces": true,
     },
+    "cmake.configureOnOpen": false,
```

Comment:
Changes from this file seems unrelated to me.
You can safely remove this from your PR.

Reply:
Will do!